src/builds/builds.service.ts

```diff
@@ -29,6 +29,25 @@ export class BuildsService {
     });
   }
 
+  async deleteOldBuilds(projectId: string, keepBuilds: number) {
+    const [buildList] = await Promise.all([
```

Comment:
is that possible to reuse `this.findMany` method here?
you could just change it like this
`async findMany(projectId: string, take?: number, skip?: number): Promise<PaginatedBuildDto> {`